pubtools/_quay/container_image_pusher.py

```diff
@@ -266,14 +270,18 @@ def push_container_images(self):
                 .get("image", {})
                 .get("sources_for_nvr", None)
             )
-            if not sources_for_nvr and not source_ml:
+            if (
+                not sources_for_nvr
+                and not source_ml
+                and not self.target_settings.get("allow_v1_containers", True)
+            ):
                 raise BadPushItem(
                     "Push item '{0}' contains a single-arch image that's not a "
                     "source image. This use-case is not supported".format(item)
                 )
             # Source image
-            if sources_for_nvr:
-                self.copy_source_push_item(item)
+            if sources_for_nvr and source_ml or not source_ml:
```

Comment:
What???????

Reply:
it's source with manifest list or non-source without it

Reply:
This condition is always true

Reply:
how's that? Its (sources_for_nvr and source_ml) or not source_ml